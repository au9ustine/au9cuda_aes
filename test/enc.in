2b7e151628aed2a6abf7158809cf4f3c
3243f6a8885a308d313198a2e0370734

000102030405060708090a0b0c0d0e0f
00112233445566778899aabbccddeeff

