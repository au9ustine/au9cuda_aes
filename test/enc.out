FFFB588C90909090909090909090909053E58955F24EC838B1055BE74E8084581FFFB7055CD1C324548900C558B10C245489528B108B2444C7F44
3925841D02DC09FBDC118597196A0B32
